web_console/about-administrator-perspective.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
```

Comment:
I am confused what this assembly is for since you have a concept module explaining the admin perspective.